### DISCUSSION TITLE
Automatically Add a Clone to the Daily Journal on Note Creation under Existing Template

### DISCUSSION THREAD

**Discussion by jasp106** [TriliumNext/Trilium#7325] [Q&A]
### Describe feature

I need assistance with implementing a function that automatically creates a clone in today's journal entry whenever a child note is created under a specific template. Essentially, I have several templates, and whenever I create a child note under any of these templates, I want a clone to today's journal entry to be automatically generated.

**Comment by meichthys** [comment]
This repository is in maintenance mode, so no future requests like this are likely to be implemented. If you would like to follow future development, check out the new community repository here: https://github.com/TriliumNext/Notes
